<ui version="4.0" >
 <class>RunnerWidget</class>
 <widget class="QWidget" name="RunnerWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>200</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Session:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="sessionLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="status" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>File progress:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QProgressBar" name="fileProgress" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_2b" >
     <property name="text" >
      <string>Session progress:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QProgressBar" name="sessionProgress" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="overallProgressLabel" >
     <property name="text" >
      <string>Overall progress:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QProgressBar" name="overallProgress" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QPushButton" name="detailsButton" >
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="QTextEdit" name="output" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
